transport/http_json/http_json.go

```diff
@@ -14,6 +14,37 @@ import (
 	"golang.org/x/net/context"
 )
 
+const requestInfoKey = "http_json_info"
+
+type requestInfo struct {
+	UseCache bool
+	UseEtag  bool
+}
+
+func fromContext(ctx context.Context) (info *requestInfo, ok bool) {
+	if val := ctx.Value(requestInfoKey); val != nil {
+		info, ok = val.(*requestInfo)
+	}
+	return
+}
+
+func newContext(parent context.Context, info *requestInfo) context.Context {
+	return context.WithValue(parent, requestInfoKey, info)
+}
+
+func DisableCache(ctx context.Context) {
```

Comment:
I think, by default cache must be disabled. You can enable it from handler. And name CacheInTransport looks better